src/SymbolDirectories/include/SymbolDirectories/Manager.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOL_DIRECTORIES_MANAGER_H_
+#define SYMBOL_DIRECTORIES_MANAGER_H_
+
+#include <filesystem>
+#include <vector>
+namespace orbit_symbol_directories {
```

Comment:
Nit: An empty line for separation would be nice

Reply:
I actually saw the same thing right before I created this PR, then I forgot to force push one last time. Anyways.. done now